src/PrestaShopBundle/Twig/Component/Toolbar.php

```diff
@@ -41,8 +41,8 @@ class Toolbar
     public string $table;
     public bool|string $helpLink;
     public bool $enableSidebar;
-    public int $currentTabLevel;
-    public array $tabs;
+    public int $currentTabLevel = 0;
+    public array $navigationTabs = [];
```

Comment:
It's ok for this one but I noticed you and maybe @boherm as well kept a lot of public fields in the component classes in previous PRs

We shouldn't have public properties on these components except for variables that we absolutely have to inject/override in the component So mostly only variables that will come from twig and that the controller injected I guess

All the other public properties should be transformed in getter functions, and ideally they should only be populated Just In Time in the getter itself Except for "advanced" variables that depend on each other in which case the mount approach may be more adapted